src/deep_clone.ts

```diff
@@ -51,10 +47,17 @@ function parse_parcelable(data, class_name)
     return Java.cast(obj, original_cls);
 }
 
-function clone_parcelable(obj)
-{
+function clone_parcelable(obj: Java.Wrapper) {
     let class_name = obj.getClass().getName();
     let obj_data = serialize_parcelable(obj);
 
     return parse_parcelable(obj_data, class_name);
 }
+
+export {
+    clone_clonable,
+    clone_serializable,
+    clone_parcelable,
+    serialize_parcelable,
```

Comment:
i'd like this repo to be splitted to subpackages/namespaces (idk how simple will it be to do so), such that we have JavaUtils namespace, and it has a ParcelableUtils module and CloneUtils (and the CloneUtils' funcs that clones parcelables will use ParcelableUtils' funcs)